### ISSUE TITLE
Entry in File List

### ISSUE THREAD

**Issue by Rimole** [ho-tex/catchfile#2]
May there be a file Test.In.tex

```
\ProvidesFile{TestIn.tex}[2023-09-12 v1.0 Test input file]
\message{Example.}
```

May there be a document

```
\documentclass{article}
\usepackage{catchfile}
\CatchFileDef{\test}{"TestIn.tex"}{}
\listfiles
\begin{document}
\end{document}
```

then the logfile contains in the `*File List*`:

```
   iftex.sty    2022/02/03 v1.0f TeX engine tests
 "TestIn.tex"
```

Even if

```
\documentclass{article}
\usepackage{catchfile}
\CatchFileDef{\test}{TestIn.tex}{}
\listfiles
\begin{document}
\end{document}
```

it has the same list entry. Either the quotes and suppressing `\Provides...` is a feature, which should be in the documentation, or it is a (minor) bug.

**Comment by u-fischer** [comment]
It has nothing to do with the content. That is not read as the content is simply stored in a macro. 
If you do `\CatchFileDef{\test}{"test.bib"}{}`, you will get `"test.bib"`. 

And yes the quotes are a small bug, LaTeX removes them but catchfile didn't catch up with recent changes in the handling of files with spaces here.

**Comment by Rimole** [comment]
@u-fischer Yes, but even `\CatchFileDef{\test}{TestIn.tex}{}` results in `"TestIn.tex"` in the file list in the log file. Might even be useful as reminder: That file was not input, but `\CatchFileDef`ined (or `\CatchFileEdef`ined) with the catchfile package. So, if we call it a feature, it would be nice to mention it in the manual. It is no problem, it was just "Why do I have files with quotation marks in the file list?!"
